person.rb

```diff
@@ -29,4 +29,8 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @Joel-100 great work on your association, however, your add_rental method needs to include the date, self, and person as parameters to create a new rental. I also noted you have not initialized rentals which will be used in this method